charts/tsuru-api/values.yaml

```diff
@@ -22,6 +22,7 @@ strategy:
 
 ingress:
   enabled: false
+  ingressClassName: "nginx"
```

Comment:
@reginaldosousa could you let this value empty to keep the current behavior ?

Reply:
Fixed